packages/frontend/src/pages/CreateProject.tsx

```diff
@@ -1,88 +1,102 @@
-import { WarehouseTypes } from '@lightdash/common';
-import React, { FC, useEffect, useState } from 'react';
+import { formatDate, WarehouseTypes } from '@lightdash/common';
+import moment from 'moment';
+import { FC, useEffect } from 'react';
+import { Redirect, useParams } from 'react-router-dom';
 import Page from '../components/common/Page/Page';
 import PageSpinner from '../components/PageSpinner';
-import { CreateProjectConnection } from '../components/ProjectConnection';
-import ConnectionOptions from '../components/ProjectConnection/ProjectConnectFlow/ConnectionOptions';
-import HowToConnectDataCard from '../components/ProjectConnection/ProjectConnectFlow/HowToConnectDataCard';
-import WareHouseConnectCard from '../components/ProjectConnection/ProjectConnectFlow/WareHouseConnectCard.tsx';
+import ConnectManually from '../components/ProjectConnection/ProjectConnectFlow/ConnectManually';
+import ConnectSuccess from '../components/ProjectConnection/ProjectConnectFlow/ConnectSuccess';
+import ConnectUsingCLI from '../components/ProjectConnection/ProjectConnectFlow/ConnectUsingCLI';
 import { ProjectFormProvider } from '../components/ProjectConnection/ProjectFormProvider';
 import { useOrganisation } from '../hooks/organisation/useOrganisation';
+import { useCreateAccessToken } from '../hooks/useAccessToken';
+import useUser from '../hooks/user/useUser';
+import useSearchParams from '../hooks/useSearchParams';
 import { useApp } from '../providers/AppProvider';
-import {
-    BackToWarehouseButton,
-    CreateHeaderWrapper,
-    CreateProjectWrapper,
-    Title,
-} from './CreateProject.styles';
 
 export type SelectedWarehouse = {
     label: string;
     key: WarehouseTypes;
     icon: string;
 };
 
+enum ConnectMethod {
+    CLI = 'cli',
+    MANUAL = 'manual',
+}
+
 const CreateProject: FC = () => {
-    const { health } = useApp();
-    const { isLoading, data: orgData } = useOrganisation();
-    const [selectedWarehouse, setSelectedWarehouse] = useState<
-        SelectedWarehouse | undefined
-    >();
-    const [hasDimensions, setHasDimensions] = useState<string>();
+    const { isLoading: isLoadingOrganisation, data: organisation } =
+        useOrganisation();
+
+    const {
+        health: { data: health, isLoading: isLoadingHealth },
+    } = useApp();
+
+    const { data: user, isLoading: isUserLoading } = useUser(
+        !!health?.isAuthenticated,
+    );
+
+    const {
+        mutate: mutateAccessToken,
+        data: tokenData,
+        isLoading: isTokenLoading,
+    } = useCreateAccessToken();
```

Comment:
I have to change this and need help.
I have two options:
1. check if there are tokens named CLI and reuse it without generating new ones every time.
2. check if user has already authenticated with CLI and do not show `lightdash login` step at all (how do I detect if user authenticated using CLI??)

Reply:
> 1. check if there are tokens named CLI and reuse it without generating new ones every time.

Most of the apps only show the token on creation and can't be accessed after that for security reasons. 
I have seen some apps that allow users to see an old token but need to re-enter the email/password. This is to avoid session hijacks. So this would break the simplicity of the flow. 

> 2. check if user has already authenticated with CLI and do not show lightdash login step at all (how do I detect if user authenticated using CLI??)

Simplest solution is to check if the user already has a token. If so, we can assume they used it in the CLI before.